File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions

service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    wait = WebDriverWait(driver, 10)

    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    # Print the title of the page
    print("Page title is: ", driver.title)

    confirm_alert = driver.find_element(By.ID,"confirm")
    confirm_alert.click()

    alert = driver.switch_to.alert

    print(alert.text)

    alert.accept()

    # alert.dismiss()

    driver.close()

File: Selenium/Activities/Activity19.java
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import java.time.Duration;

public class Activity19 {
    public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));


        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("Page title is: "+driver.getTitle());

        WebElement confirm_alert = driver.findElement(By.id("confirm"));
        confirm_alert.click();

        Alert alert = driver.switchTo().alert();

        System.out.println(alert.getText());

        alert.accept();

//             alert.dismiss();

        driver.close();
}
}
